pages/common/nmap.md

```diff
@@ -4,6 +4,16 @@
 > Some features only activate when Nmap is run with privileges.
 > More information: <https://nmap.org>.
 
+- If Nmap throws a root privileges error use it with `sudo` or `admin rights cmd`
+
+- To check if the IP address `192.168.1.101` is up, with it’s operating system info. It may show RTTVAR logs (for TCP) for a long time:
```

Comment:
Perhaps this will read better?
```suggestion
- Check if a given IP address is up, and attempt to get the remote operating system:
```

Reply:
This is better:
```suggestion
- Check if an IP address is up, with OS info:
```

Reply:
Hrm, I'm not keen on shortening `OS info` there, and it's also ambiguous what it does with it. What about this?
```suggestion
- Check if an IP address is up, and guess the remote host's operating system:
```